lib/roo_on_rails/rack/populate_env_from_jwt.rb

```diff
@@ -0,0 +1,98 @@
+require 'json/jwt'
+require 'faraday'
+require 'faraday_middleware'
+
+module RooOnRails
+  module Rack
+    class PopulateEnvFromJWT
+      UnnacceptableKeyError = Class.new(RuntimeError)
+      VALID_JWK_URL_PREFIXES = [
+        'https://www.deliveroo.com/identity-keys/',
+        'https://identity.deliveroo.com/identity-keys/'
```

Comment:
Identity is on `identity.deliveroo.net` at the moment. Also, you'll need some way to make this configurable for staging so that `identity-staging.deliveroo.net` will work in that environment.

Reply:
Hmm, yeah, I was thinking about environments (see the `.dev` stuff below). I'll add per-environment hash here, which isn't ideal, but it'll get the job done.